frontend/public/components/modals/_modals.scss

```diff
@@ -40,6 +40,7 @@
   // so that input, textarea, button, and input-group-addon don't mask the inner scroll shadows
   input, textarea {
     &.pf-c-form-control {
+      max-width: 100%;
```

Comment:
Nit.  We aim to alphabetize declarations by property in the console code base, so move this line after the background-color declaration below.

Reply:
Okay. It has been moved as required.